stemcell_builder/stages/system_google_packages/assets/instance_configs.cfg.template

```diff
@@ -1,3 +1,10 @@
-'InstanceSetup': {
-    'set_host_keys': 'false',
-},
+[Daemons]
+accounts_daemon = true
+clock_skew_daemon = true
+ip_forwarding_daemon = true
+[InstanceSetup]
+network_enabled = true
+optimize_local_ssd = true
+set_host_keys = false
```

Comment:
Prevent the Google daemon from setting SSH host keys, which races with BOSH's own configuration.